clients/devtools-entry.js

```diff
@@ -29,7 +29,7 @@ function createConfig(categoryIDs, device) {
   const settings = {
     onlyCategories: categoryIDs,
     // DevTools handles all the emulation
-    screenEmulation: false,
+    screenEmulation: {disabled: true},
```

Comment:
the type error annotation raises an annoyingly good point that `disabled: true` still requires the other properties to be set 😞

Reply:
thx done.